homepage/homepage/content/docs/using-covalues/version-control.mdx

```diff
@@ -0,0 +1,240 @@
+import { CodeGroup, ContentByFramework } from "@/components/forMdx";
+import { Alert } from "@garden-co/design-system/src/components/atoms/Alert";
+
+export const metadata = {
+  description: "Learn how to use branching and merging in Jazz for collaborative editing with conflict resolution and private branches."
+};
+
+# Version Control
+
+Jazz provides built-in version control through branching and merging, allowing multiple users to work on the same CoValue simultaneously without conflicts. This enables collaborative editing workflows where users can create private branches, make changes, and merge them back to the main version.
+
+## Working with branches
+
+### Creating Branches
+
+To create a branch, use the `unstable_branch` option when loading a CoValue:
+
+<CodeGroup>
+```ts
+const branch = await Project.load(projectId, {
+  unstable_branch: { name: "feature-branch" }
+});
+```
+</CodeGroup>
+
+<ContentByFramework framework={["react", "react-native", "expo"]}>
+You can also create a branch via the `useCoState` hook:
+
+<CodeGroup>
+```ts
+const branch = useCoState(Project, projectId, {
+  unstable_branch: { name: "feature-branch" }
+});
+```
+</CodeGroup>
+</ContentByFramework>
+<ContentByFramework framework={["svelte"]}>
+You can also create a branch via `CoState`:
+
+<CodeGroup>
+```ts
+const branch = new CoState(Project, projectId, {
+  unstable_branch: { name: "feature-branch" }
+});
+```
+</CodeGroup>
+</ContentByFramework>
+
```

Comment:
```suggestion

You can also include nested CoValues in your branch by using a [`resolve` query](/docs/using-covalues/subscription-and-loading#resolve-queries). All of the nested CoValues will have their own branched versions created, and you can access them in the same way you would normally access a deeply loaded property.

<Alert variant="info" className="mt-4 flex gap-2 items-center">
  In case you create a separate reference to a nested CoValue (for example by loading it by its ID), you will need to specify the branch you wish to load.
</Alert>
```